apps/web-portal-client/lib/components/dialog/ConfirmDialog.jsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import {childrenPropTypes} from '../../utils/childrenPropTypes'
+
+import {Dialog} from './Dialog'
+import {Button} from '../Button'
+
+export function ConfirmDialog({children, title, open, onClose, onConfirm}) {
+  if (!open) return null
+
+  return (
+    <Dialog open={open} onClose={onClose}>
+      <h2 className="text-xl">{title}</h2>
+      <div className="py-5">{children}</div>
+      <div className="flex justify-end">
+        <div className="p-1 w-1/5 flex">
+          <Button
+            label="No"
+            cssClass="bg-smooth-green hover:bg-smooth-green-light flex-auto"
```

Comment:
It's not the best practice to pass `custom` css to such a component like `Button`

We have to standardize button types like, for example:  `<Button type="primary" label="Yes"  />`. We can start with two types of` primary`/`default`, then component would choose to which color to provide for primary and default


Another good practice for dialogs is to highlight the acton button (in confirm dialog it's `Yes`) with primary type and `Cancel` as default type

Reply:
extracting to separate issue